Buildings/Controls/OBC/ASHRAE/PrimarySystem/BoilerPlant/Staging/SetPoints/Subsequences/FailsafeCondition.mo

```diff
@@ -0,0 +1,118 @@
+within Buildings.Controls.OBC.ASHRAE.PrimarySystem.BoilerPlant.Staging.SetPoints.Subsequences;
+block FailsafeCondition
+  "Failsafe condition used in staging up and down"
+
+  parameter Real delayEna(
+    final unit="s",
+    final displayUnit="s",
+    final quantity="Time") = 900
+    "Enable delay";
+
+  parameter Real TDif(
+    final unit="K",
+    final displayUnit="K",
+    final quantity="TemperatureDifference") = 10
+    "Required temperature difference between setpoint and measured temperature
+    for failsafe condition";
+
+  parameter Real TDifHys(
+    final unit="K",
+    final displayUnit="K",
+    final quantity="TemperatureDifference") = 1
+    "Temperature deadband for hysteresis loop"
+    annotation (Dialog(tab="Advanced"));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput TSupSet(
+    final unit="K",
+    final displayUnit="K",
+    final quantity="ThermodynamicTemperature")
+    "Hot water supply temperature setpoint"
+    annotation (Placement(transformation(extent={{-140,30},{-100,70}}),
+      iconTransformation(extent={{-140,30},{-100,70}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput TSup(
+    final unit="K",
+    final displayUnit="K",
+    final quantity="ThermodynamicTemperature")
+    "Measured hot water supply temperature"
+    annotation (Placement(transformation(extent={{-140,-70},{-100,-30}}),
+      iconTransformation(extent={{-140,-70},{-100,-30}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanOutput y
+    "Failsafe condition for chiller staging"
+    annotation (Placement(transformation(extent={{100,-20},{140,20}}),
+      iconTransformation(extent={{100,-20},{140,20}})));
+
+  Buildings.Controls.OBC.CDL.Continuous.Add add2(
+    final k2=-1,
+    y(
```

Comment:
we don't carry the unit around between input and output.

Reply:
I was using this intermediate "add" block in the validation plot, and I thought it would help to have the temperature units display. Will remove it if it is not allowed

Reply:
It's not allowed... due to translation needs. We're trying to keep it simple. Just plot both inputs in the validation plots.